README.md

```diff
@@ -29,15 +29,15 @@ This is the main script used to generate the `README.md` files for each image. T
 
 This is used by us to push the actual content of the READMEs to the Docker Hub as special access is required to modify the Hub description contents.
 
-## `generate-dockerfile-links-partial.sh`
+## `.template-helpers/generate-dockerfile-links-partial.sh`
 
 This script is used by `update.sh` to create the "Supported tags and respective `Dockerfile` links" section of each generated `README.md` from the information in the [official-images `library/` manifests](https://github.com/docker-library/official-images/tree/master/library).
 
-## `generate-repo-stub-readme.sh`
+## `.template-helpers/generate-repo-stub-readme.sh`
```

Comment:
Yeah, probably best to move it up by `update.sh` since it's supposed to be "user-facing", so to speak.